worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -317,9 +318,7 @@ final private[worker] class StorageManager(conf: CelebornConf, workerSource: Abs
         }
 
         fileInfos.computeIfAbsent(shuffleKey, newMapFunc).put(fileName, fileInfo)
-        hdfsWriters.synchronized {
-          hdfsWriters.add(hdfsWriter)
-        }
+        hdfsWriters.put(fileName, hdfsWriter)
```

Comment:
current hdfsWriters and workingDirWriters use different map key, can we align this?

Reply:
hdfswriters role is  just as workingDirWriters's value. so no need align